pallets/gbio/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::{
+	decl_error, decl_event, decl_module, decl_storage, dispatch::DispatchResult, ensure
+};
+use frame_system::ensure_signed;
+
+pub trait Config: frame_system::Config {
+	type Event: From<Event<Self>> + Into<<Self as frame_system::Config>::Event>;
+}
+
+decl_storage! {
```

Comment:
I just want to remind us if we could write FRAME v2 syntaxes to follow Substrate 3.0 evolvement.
https://substrate.dev/docs/en/knowledgebase/runtime/pallets